### PR TITLE
UX Pattern Library changes

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd

### CODE REVIEW

**Comment by stvstnfrd** [comment]
Do we need to bump the version number too?

**Comment by caesar2164** [comment]
Hmm, we could, but do we need to since we install from master branch?

**Comment by stvstnfrd** [comment]
@caesar2164 If we install from master, let's not merge this and instead, update the platform to install a specific version (starting w/ this one). Else we'd be rolling the change out at the next deploy.

**Comment by caesar2164** [comment]
@stvstnfrd shouldn't we do both? merge to master AND change installation from a specific version?

**Comment by stvstnfrd** [comment]
@caesar2164 Sure, you'll need to tag the current master first and submit a pr to platform master to install from that tag. After that, you can merge this.

File: package.json
Comment (stvstnfrd) [inline package.json:3]: Why `.3`? Isn't this our first (stanford) release of `0.18.2`?

File: pattern-library/sass/global/_stanford.scss
Comment (stvstnfrd) [inline pattern-library/sass/global/_stanford.scss]: Trailing space before `;`.

**Review by stvstnfrd — CHANGES_REQUESTED** [review]

File: package.json
Comment (caesar2164) [reply package.json:3]: because we already have other tags for `0.18.2` that don't have the requisite code... I was silly and created tags too early...

**Review by stvstnfrd — APPROVED** [review]

**Pull request merged**
